extension-readme.md

```diff
@@ -1,182 +1,242 @@
+
+   ![Create SAM App 2](./resources/marketplace/create-sam-app-still-2.png)
+
+{ [Return to Developing](#developing-serverless-applications) } or { [Return to Usage](#usage) }
 
-The Toolkit produces the application code and files, adds them to the location you specified, and opens the program file in the editor.
+`________________`
 
-For more information, see [Creating a Serverless Application](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/create-sam.html) in the _AWS Toolkit for Visual Studio Code User Guide_.
+### <a id="run-debug-serverless-application"></a>**Step 2**: Run or Debug the Serverless Application
 
-[Return to Top](#top)
+For complete instructions, see [Run and Debug Your Serverless Application](https://docs.aws.amazon.com/console/toolkit-for-vscode/create-sam#run-debug-sam-app) in the _AWS Toolkit for Visual Studio Code User Guide_.
 
-#### <a id="run-debug-serverless-application"></a>Run or Debug a Serverless Application
+In summary:
+1. Select the **AWS** icon in the Activity Bar, ![AWS Explorer Icon](./resources/marketplace/aws-explorer-icon.png), to ensure that the AWS Toolkit for Visual Studio Code is running.
+1. Select the Project icon in the Activity Bar, ![Project Explorer Icon](./resources/marketplace/project-explorer-icon.png), go to the `hello-world` directory, and open `app.js`.
+1. Select the **Run Locally** CodeLens. (Or, if needed, first select the **Configure** link from the Codelenses, enter event data in `templates.json`, save, and _then_ select the **Run Locally** CodeLens.)
 
-![Configure and Run](./resources/marketplace/sam-configure-and-run.gif)
+   ![Configure and Run 1](./resources/marketplace/sam-configure-and-run-still-1.png)
 
-(_Clip is time lapsed_)
+1. Note the run status and results in the **OUTPUT** panel.
 
-After you choose the **AWS** icon in the Activity Bar, CodeLenses display within open serverless application code files above functions that use AWS Lambda function handler syntax. (A _handler_ is a function that Lambda calls to start execution of a Lambda function.) These CodeLenses enable you to run or debug the corresponding serverless application locally. CodeLens actions here include:
+   ![Configure and Run 1](./resources/marketplace/sam-configure-and-run-still-2.png)
+
+#### Additional Information
+After you choose the **AWS** icon in the Activity Bar, CodeLenses appear in serverless-application code files above certain functions. The functions that have these Codelenses are those that use AWS Lambda-function handler syntax. A _handler_ is a function that Lambda calls to start execution of a Lambda function. These CodeLenses enable you to locally run or debug the corresponding serverless application. CodeLens actions in the Toolkit include:
 
 * **Configure**, for specifying function configurations such as an event payload and environment variables.
```

Comment:
Do we have any documentation on how to configure through this menu? If so, we definitely need to link to it, it's not the most intuitive in its current state.

Reply:
We allude to it (here and in the user guide), but I haven't found any documentation that digs into it more deeply.
The documentation might be out there, but I just don't know about it.
Good idea to add some somewhere eventually.

Reply:
We can follow up with this in the docs. When we get that in there, we'll probably want a link from this guide to that page.